### PR TITLE
Add globbing pattern to cli-tooling.adoc

### LINKED ISSUE #38545: Enhance Adding extension section in cli-tooling documentation page
### Description

Working on the issue #38542 was possible to see that [this documentation page](https://quarkus.io/guides/cli-tooling#adding-extensions) does not comment about to install an extension using globbing pattern:

```
quarkus extension add smallrye-*
```

If it makes sense, I created a draft pull request for this.

### Implementation ideas

Add globbing pattern into the documentation.

### PR DESCRIPTION
**Pull request by mcruzdev**
Fixes #38545

### CODE REVIEW

File: docs/src/main/asciidoc/cli-tooling.adoc [hunk 1]
```diff
@@ -476,6 +476,12 @@ quarkus ext add kubernetes health
 [SUCCESS] ✅ Extension io.quarkus:quarkus-smallrye-health has been installed
 ----
 
+You can install all extensions which match a globbing pattern:
```
Comment (gsmet) [inline docs/src/main/asciidoc/cli-tooling.adoc:479]:
```suggestion
You can install all extensions which match a glob pattern:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks for this addition.

I added a small suggestion as `glob pattern` seems to be the common usage.

Could you adjust and squash? Thanks!

**Comment by mcruzdev** [comment]
Hi @gsmet, thank you for reviewing! I updated, done!

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**